### PR TITLE
Clarify changed semantics of http::server::getSocketDescriptor()

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 # git master
 
+* [185](https://github.com/HBPVIS/ZeroEQ/pull/185):
+  http::server::getSocketDescriptor() notifies on write after
+  [#184](https://github.com/HBPVIS/ZeroEQ/pull/184).
```
Comment (rdumusc) [inline doc/Changelog.md]: Mistake with this line?

File: zeroeq/http/server.h [hunk 2]
```diff
@@ -82,7 +82,8 @@ class Server : public zeroeq::Receiver
      * Get the underlying socket descriptor.
      *
      * Can be used by client code to be notified when new data is available and
-     * subsequently call receive.
+     * subsequently call receive. Due to implementation details, the socket
+     * notifies on write.
```
Comment (rdumusc) [inline zeroeq/http/server.h]: I would have emphasized more:
notifies on write, not on read.

File: doc/Changelog.md (hunk 1 above)
Comment (tribal-tec) [inline doc/Changelog.md]: no, just putting a link to the pull request that introduced the change of semantics. Or what do you mean?

Comment (rdumusc) [inline doc/Changelog.md]: oh ok sorry, I misread it as a duplicate of the previous line

**Pull request merged**